twitter/settings.py

```diff
@@ -129,6 +129,9 @@
 # https://docs.djangoproject.com/en/3.2/howto/static-files/
 
 STATIC_URL = '/static/'
+STATICFILES_DIRS = [
+    BASE_DIR / "static"
+]
```

Comment:
I am not 100% sure if this is a valid way to do things here.

Reply:
yes I did this in training

![image](https://github.com/GaganDureja/twitter/assets/53633318/e4017c0c-828d-4cce-b043-a4215f68ceb0)


https://docs.djangoproject.com/en/5.0/howto/static-files/